www.foia.gov/sitemap.html

```diff
@@ -1,131 +1,131 @@
-                <li><a href="report-makerequest.html">Where to Make a FOIA Request</a></li>
-                <li><a href="opengov.html">Open Government</a></li>
-                <li><a href="faq.html">Frequently Asked Questions</a></li>
-			  </ul>
-            </li>
-            <li><a href="sitemap.html">Site Map</a></li>
-			<li><a href="accessibility.html">Accessibility</a></li>
-			<li><a href="foia-info.html">FOIA</a></li>
-			<li><a href="privacy.html">Privacy Policy</a></li>
-			<li><a href="legal.html">Policies &amp; Disclaimers</a></li>
-			<li><a href="quality.html">Information Quality</a></li>
-			<li><a href="developer.html">Developer Resources</a></li>
-            <li><a href="news.html">News</a>
-                <ul>
-                  <li><a href="news-spotlight.html">News Spotlight</a></li>
-                  <li><a href="news-archive.html">News Archive</a></li>
-                </ul>
-            </li>
-            <li><a href="glossary.html">Glossary</a></li>
-            <li><a href="feedback.html">Feedback</a></li>
-          </ul>
-          <p align="right">Last Updated: 6/2011</p>
-          <p align="right"><a href="sitemap.html#top">Return to Top</a></p>
-          
-        </div>
-        <div class="clear"></div>
-      </div>
-      <!-- End Content -->
-    </div>
-    <!-- End Main Page -->
-    <!-- Start Footer -->
-    <div id="footer"><a href="index.html"><img src="images/foia-footer.gif" alt="foia.gov" /><span>Foia.gov</span></a>
-  <div id="footerNav">   
-	<a id="footerLinks"></a>
-  <ul>
-    <li><a href="sitemap.html">Site Map</a></li>
-	<li><a href="https://www.justice.gov/accessibility/accessibility-information">Accessibility</a></li>
-	<li><a href="foia-info.html">FOIA</a></li>
-	<li><a href="https://www.justice.gov/doj/privacy-policy">Privacy Policy</a></li>
-	<li><a href="https://www.justice.gov/legalpolicies">Policies &amp; Disclaimers</a></li>
-	<li><a href="quality.html">Information Quality</a></li>
-	<li><a href="developer.html">Developer Resources</a></li>
-	<li><a href="http://www.justice.gov">Justice.gov</a></li>
-	<li><a href="http://www.usa.gov">USA.gov</a></li>
-  </ul>
-  </div>
-  <div class="clear">&nbsp;</div>
-  <div align="center">
-    <a href="http://www.data.gov/"><img src="images/datagov-footer.gif" alt="data.gov" /><span>Data.gov</span></a>
-	<a href="http://www.whitehouse.gov/open"><img src="images/opengov-footer.gif" alt="whitehouse.gov/open" /><span>Whitehouse.gov/open</span></a>
-	<a href="http://www.usaspending.gov/"><img src="images/usaspending-footer.gif" alt="usaspending.gov" /><span>USASpending.gov</span></a>
-    <a href="http://www.recovery.gov/"><img src="images/recoverygov-footer.gif" alt="recovery.gov" /><span>Recovery.gov</span></a>
-  </div>
-</div>
-    <!-- End Footer -->
-  </div>
-</div>
-<!-- End FOIA Template -->
-</body>
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN" "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
```

Comment:
I'm guessing we didn't jekyllize this one because the sitemap is going to change anyway.

Reply:
Yes, but this PR isn't even complete. There are still some pages that need to be migrated, they are lower priority but we will get to them later.